docs/src/test/scala/docs/http/scaladsl/server/directives/JsonStreamingExamplesSpec.scala

```diff
@@ -18,31 +18,29 @@ import scala.concurrent.Future
 
 class JsonStreamingExamplesSpec extends RoutingSpec {
 
-  //#models
+  //#tweet-model
   case class Tweet(uid: Int, txt: String)
-  case class Measurement(id: String, value: Int)
```

Comment:
Moving this `Measurement` case class to the place where it is explained in detail.